### ISSUE TITLE
autoRecoveryDaemonEnabled keeps getting added to configuration

### ISSUE THREAD

**Issue by maddisondavid** [pravega/pravega#1464]
**Problem description**
Issue 1457 introduced a small bug where the `autoRecoveryDaemonEnabled` line would keep getting added to the end of the configuration if the script was rerun

**Problem location**
`/docker/bookkeeper/entrypoint.sh`

**Suggestions for an improvement**
Remove existing autoRecoveryDaemonEnabled setting before adding new one

**Comment by adrianmo** [comment]
This issue was resolved in #3192 with the following fix to prevent the option to be added multiple times.

```
sed -i "s|# autoRecoveryDaemonEnabled=.*\$|autoRecoveryDaemonEnabled=${BK_AUTORECOVERY}|" ${BK_HOME}/conf/bk_server.conf
```

**Comment by fpj** [comment]
Closing this issue, please re-open if it manifests again.
